packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -1019,6 +1042,107 @@ class ThemeData extends Diagnosticable {
     properties.add(DiagnosticsProperty<ColorScheme>('colorScheme', colorScheme, defaultValue: defaultData.colorScheme));
     properties.add(DiagnosticsProperty<DialogTheme>('dialogTheme', dialogTheme, defaultValue: defaultData.dialogTheme));
     properties.add(DiagnosticsProperty<Typography>('typography', typography, defaultValue: defaultData.typography));
+    properties.add(DiagnosticsProperty<CupertinoThemeData>('cupertinoOverrideTheme', cupertinoOverrideTheme, defaultValue: defaultData.cupertinoOverrideTheme));
+  }
+}
+
+/// A [CupertinoThemeData] that defers unspecified theme attributes to an
```

Comment:
to its Material [Theme] ancestor.

Reply:
Not 100% technically true. It's not a widget so it has no ancestors. Material Theme builds it with a material's ThemeData.